python/lsst/obs/sdss/sdssNullIsr.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
```

Comment:
I think we're supposed to have this line for emacs compatibility, though I can't find anything in the python style guide explicitly saying so.